src/repository/screens/repository.screen.js

```diff
@@ -266,6 +302,16 @@ const styles = StyleSheet.create({
   listTitle: {
     color: colors.black,
     fontFamily: 'AvenirNext-Medium'
+  },
+  overlay: {
```

Comment:
looks like this isn't being used at all, most likely forgot to remove it when you created a separate component :)